pkg/serverless/invocationlifecycle/trace_test.go

```diff
@@ -87,151 +75,292 @@ func TestInjectSpanIDWithContext(t *testing.T) {
 	assert.Equal(t, uint64(1234), currentExecutionInfo.SpanID)
 }
 
-func TestStartExecutionSpanWithoutPayload(t *testing.T) {
-	currentExecutionInfo := &ExecutionStartInfo{}
-	lp := &LifecycleProcessor{
-		requestHandler: &RequestHandler{
-			executionInfo: currentExecutionInfo,
+func TestStartExecutionSpan(t *testing.T) {
```

Comment:
All tests previously in this file testing the `startExecutionSpan` method were removed in favor of this single parameterized test.